jetty-security/src/main/java/org/eclipse/jetty/security/ConstraintSecurityHandler.java

```diff
@@ -67,7 +71,8 @@
     private final List<ConstraintMapping> _constraintMappings = new CopyOnWriteArrayList<>();
     private final List<ConstraintMapping> _durableConstraintMappings = new CopyOnWriteArrayList<>();
     private final Set<String> _roles = new CopyOnWriteArraySet<>();
-    private final PathMap<Map<String, RoleInfo>> _constraintMap = new PathMap<>();
+
+    private final PathMappings<Map<String, RoleInfo>> _constraintMap = new PathMappings<>();
```

Comment:
The change to PathMappings has already been done in jetty-10, along with a rename of _constraintMap to _constraintRoles, so make it easy on yourself and do the same rename here in jetty-9.